plugins/tls_cert

```diff
@@ -3,15 +3,13 @@
 use strict;
 use warnings;
 
-# Very basic script to create TLS certificates for qpsmtpd
+# create TLS certificates for qpsmtpd
 use File::Temp qw/ tempfile tempdir /;
 use Getopt::Long;
+use Sys::Hostname;
```

Comment:
FTR, it seems Sys::Hostname::hostname() at least potentially uses gethostbyname():
```
sub hostname {

  # method 1 - we already know it
  return $host if defined $host;

  # method 1' - try to ask the system
  $host = ghname() if defined &ghname;
  return $host if defined $host;

  if ($^O eq 'VMS') {

    # method 2 - no sockets ==> return DECnet node name
    eval { local $SIG{__DIE__}; $host = (gethostbyname('me'))[0] };
```

This doesn't seem any more remotely exploitable than the 'hostname' command, though